buster/completers/base.py

```diff
@@ -0,0 +1,112 @@
+import logging
```

Comment:
add an `__init__.py` in this folder, and import the various completers in it, so we can  just do `from buster.completers import get_completer`.

Reply:
This will be useful when the structure inside of completers will change, as nothing outside of it will need to change.

Reply:
good idea, done